test/albumController.test.js

```diff
@@ -173,24 +159,130 @@ describe('AlbumController API Tests', () => {
 
     await Album.insertMany(albumsToCreate);
 
-    // Make a GET request to the /api/v1/albums/filter endpoint with a limit parameter set to 4
     const limit = 3;
     const response = await request(app).get(
       `/api/v1/albums/filter?limit=${limit}`
     );
+    expect(response.status).toBe(StatusCodes.BAD_REQUEST);
```

Comment:
Why do we expect it the response to be a `BAD_REQUEST` code here and then in the next line an `OK` ?

Reply:
Removed bad request status , commit 2cb2ecd237dc13b8d6c8b3c04eaddcac5b6345a6